.gitignore

```diff
@@ -43,3 +43,10 @@ build/config/buildinfo.properties
 
 www-built
 www-ghdeploy
+/node_modules/
+/tmp/
+/scripts/kiwix-*.pem
+/scripts/set_secret_environment_variables.sh
```

Comment:
Oups, I thought they had been exposed, so I revoked and re-created them.
No matter : it doesn't hurt.